internal/configuration/configuration_test.go

```diff
@@ -28,14 +29,14 @@ func Test_configuration_settings(t *testing.T) {
 	var testConfigs = []config{
 		{"http://localhost", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{}},
 		{"https://localhost", "https://localhost", "https://localhost", "https://localhost", "token-1", "token-2", "10", "15", "15", []string{}},
-		{"", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_OPENSHIFT_API_URL needs to be a valid URL."}},
-		{"foo", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_OPENSHIFT_API_URL needs to be a valid URL."}},
-		{"/foo", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_OPENSHIFT_API_URL needs to be a valid URL."}},
-		{"ftp://snafu", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_OPENSHIFT_API_URL needs to be a valid URL."}},
-		{"", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_OPENSHIFT_API_URL needs to be a valid URL."}},
-		{"http://localhost", "", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_JENKINS_PROXY_API_URL needs to be a valid URL."}},
-		{"http://localhost", "http://localhost", "", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_F8TENANT_API_URL needs to be a valid URL."}},
-		{"http://localhost", "http://localhost", "http://localhost", "", "token-1", "token-2", "10", "15", "15", []string{"Value for JC_TOGGLE_API_URL needs to be a valid URL."}},
+		{"", "http://localhost", "http://localhost", "http://localhost", "token-1", "token-2", "10", "15", "15", []string{"value for JC_OPENSHIFT_API_URL needs to be a valid URL"}},
```

Comment:
you just changed to start with lower case and no punctuation, right?

Reply:
yup

Reply:
Good